File: Questions/Day-5/cpp/Q2.cpp
#include <iostream>
using namespace std;

void sandglass() {
        for (int i=5; i>0; --i) {
            for (int j=i; j<5; ++j) {
                cout<<" ";
            }
            for(int j=i; j> 0; --j) {
                cout<<"* ";
            }
            cout<<endl;
        }
        for (int i=1; i<=5; ++i) {
            for (int j=i; j<5; ++j) {
                cout<<" ";
            }
            for(int j=i; j> 0; --j) {
                cout<<"* ";
            }
            cout<<endl;
        }
    }
int main()
{
    sandglass(); 
    return 0;
}

File: Questions/Day 5/java/Q2.java
public class Q2 {
    public static void main(String[] args) {
        sandglass();   
    }

    private static void sandglass() {
        for (int i=5; i>0; --i) {
            for (int j=i; j<5; ++j) {
                System.out.print(" ");
            }
            for(int j=i; j> 0; --j) {
                System.out.print("* ");
            }
            System.out.println();
        }
        for (int i=1; i<=5; ++i) {
            for (int j=i; j<5; ++j) {
                System.out.print(" ");
            }
            for(int j=i; j> 0; --j) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}
